exonum-java-binding/core/rust/src/utils/mod.rs

```diff
@@ -27,3 +27,20 @@ pub use self::errors::{
     panic_on_exception, unwrap_exc_or, unwrap_exc_or_default, unwrap_jni, unwrap_jni_verbose,
 };
 pub use self::jni::{get_class_name, get_exception_message};
+
+/// Asserts that given closure panics while executed and the resulting error message contains given
+/// substring.
+#[cfg(test)]
+pub fn assert_panics<F, R>(err_substring: &str, f: F)
```

Comment:
I remember we had this method somewhere — was it lost with some obsolete native ITs?

Reply:
Yes, it's gone within ECR-3480 with obsolete service/transactions proxy tests afair.

Reply:
Lol, deja vu: https://github.com/exonum/exonum-java-binding/pull/1231/files#diff-19687bf9dc0b70f9c7df7e63d6a5d2dbR646